discovery-client/build.gradle

```diff
@@ -1,24 +1,24 @@
 plugins {
     id("io.micronaut.build.internal.module")
 }
+
 dependencies {
-    annotationProcessor(libs.micronaut.docs.asciidoc.config.props)
-    annotationProcessor(libs.micronaut.graal)
+    annotationProcessor(mn.micronaut.graal)
 
-    api(libs.micronaut.http.client)
-    api(libs.micronaut.validation)
+    api(mn.micronaut.http.client)
+    api(mn.micronaut.validation)
 
-    implementation(libs.reactor.core)
+    implementation(mn.reactor.core)
 
-    compileOnly(libs.micronaut.management)
+    compileOnly(mn.micronaut.management)
 
-    testAnnotationProcessor(libs.micronaut.inject.java)
-    testImplementation(libs.spock)
+    testAnnotationProcessor(mn.micronaut.inject.java)
+    testImplementation("org.testcontainers:spock")
```

Comment:
should we define  `org.test-containers:spock` in `libs.versions.toml` ?

Reply:
done here: https://github.com/micronaut-projects/micronaut-discovery-client/pull/401/commits/cc93ca7dff369cdbbf0df21b7c6c1f05bfb58d82